parmed/tools/actions.py

```diff
@@ -2927,6 +2927,16 @@ def __repr__(self):
                    len(self.parm.atoms), len(self.parm.residues))
         )
 
+        _rset = set(res.name for res in self.parm.residues).__str__() + '\n'
+        _rcount = Counter(res.name for res in self.parm.residues).__str__() + '\n'
```

Comment:
I will change to ``str( )``. I have habit to use ``.method`` to avoid using to many ``( )``, and for auto completion.